collect_app/src/main/res/layout/sort_item_layout.xml

```diff
@@ -20,7 +20,6 @@ the License.
     android:layout_height="wrap_content"
     android:clickable="true"
     android:focusable="true"
-    android:foreground="?android:attr/selectableItemBackground"
```

Comment:
Why did you remove it?

Reply:
Sorry @grzesiek2010! I was fixing it actually. I found it [here](https://stackoverflow.com/questions/52329249/why-doesnt-androidforeground-attribute-work) that `android:foreground` attribute doesn't work in API 23 or lower.